examples/quickstart/groundtruth_dataset_persistence.ipynb

```diff
@@ -24,7 +24,6 @@
    "metadata": {},
```

Comment:
<p>this is not really what you're doing here right? Aren't you just doing a regular gt feedback function (not benchmarking/meta-eval)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>no we are doing meta evaluation below, and the feedback function that's being meta-evaluated is context relevance.</p><p> </p><p>You can see I instantiate a <code>TruBenchmarkExperiment</code> in the following cells </p>